tests/client.py

```diff
@@ -72,6 +77,6 @@ def graphql_server():
 
```

Comment:
In line 75 `debug=app.debug` we are defining the parameter passed to graphql_sync based on an attribute of app, which does exist for a Flask app but not for a FastAPI app

Reply:
It exists as a field https://github.com/fastapi/fastapi/blob/master/fastapi%2Fapplications.py#L823

Reply:
In the docs is not there, but the code is the code :)